source/manual/data-gov-uk-supporting-ckan.html.md

```diff
@@ -212,8 +212,10 @@ psql ckan_production -c "SELECT id FROM harvest_source WHERE name = '[NAME]'"
 
 #### Cancelling a current job
 
-To cancel a currently running job, you will require a JOB_ID from the
-[Listing current jobs](#listing-current-jobs) section.
+Sometimes a harvest job can get stuck and not complete, and it's not possible to
+restart/reharvest through the UI. You will can get the `JOB_ID` from the
```

Comment:
There's an extra word in there`You will can get`

Reply:
Thanks @MuriloDalRi , I've updated this now!